### DISCUSSION TITLE
How to fix "Could not verify JWT: JWTExpired"

### DISCUSSION THREAD

**Discussion by PtradeLLC** [nhost/nhost#2276] [Q&A]
Hey folks,
Our Nhost dashboard continues to display `Could not verify JWT: JWTExpired` - How can it be fixed? 

I would copy/paste the code responsible for it but I don't quite know where to start as all was well until it wasn't. Plus this issue isn't on our NextJs app, its our `app.nhost.io` dashboard so this got me  staring at the screen while watching my beard turn gray.

Another observation is that `user` instance is not saved in `Hasura` database which I'm guessing is related to this issue. Please help.

<img width="1207" alt="Screenshot 2023-09-28 at 6 27 41 PM" src="https://github.com/nhost/nhost/assets/138940763/56eeac2a-1b8a-440a-9b57-5f98c61e7759">

Thanks for your suggestions.

**Comment by dbarrosop** [comment]
As you yourself mentioned the issue is in our dashboard, it has nothing to do with your service. We need to look into this but most likely the tab was left open for a long time and the access token expired and wasn't refreshed appropiately.

> Another observation is that user instance is not saved in Hasura database which I'm guessing is related to this issue. Please help.

You mean in your database? Of course not, the user you use to login to our dashboard is part of Nhost so it is in our database, which you don't have access to of course :) Your database is for your users and your data only.
